project/Dependencies.scala

```diff
@@ -100,7 +101,9 @@ object Dependencies {
   )
 
   lazy val httpCore = l ++= Seq(
-    Test.sprayJson, // for WS Autobahn test metadata
+    Compile.jwtScala,
+    Compile.sprayJson,
```

Comment:
I'm not convinced the convenience is worth it vs everyone using akka-http now getting these two transitive dependencies. I think we should make it a separate `akka-http-jwt` module that users opt in for using instead.